FRAMEWORK-OVERVIEW.md

```diff
@@ -82,7 +83,12 @@ The event delegates are the ones that add behaviour to certain events inside a s
 * [SignupControllerTransitionDelegate](Authentication/SignUp/View/SignupControllerTransitionDelegate.swift)
 
 
+## Using Custom Views
 
+If the user wants to use custom views for Login, Sign Up or both, he or she should create a class that implements `AuthenticationComponentsFactoryType` and override the correspondent method (`createLoginView(withDelegate delegate: LoginViewDelegate, loginProviders: [LoginProvider]) -> LoginViewType` for example) and pass it to the `AuthenticationCoordinator` on creation.
+The custom view will need to implement `LoginViewType` (or `SignUpViewType`). To implement that protocol the user will also need to implement `Renderable` and `LoginFormType` (or `SignUpFormType`) and that one will also need to implement `AuthenticationFormType`. 
+Note that those protocols will ask to implement properties that should match an UI element from the view (`var passwordTextField: UITextField` for example). In those cases the best solution is probably to return an IBOutlet reference of the element.
+If the user is using `GoogleLoginProvider`, the ViewController will need to implement the protocol `GIDSignInUIDelegate` and in `viewDidLoad` add the line `GIDSignIn.sharedInstance().uiDelegate = self`.
```

Comment:
But shouldn't you use the controller that the framework gives you?
Right now, there no way to safely replace the default controller, that's why we give no open chance to do that.

Reply:
ah you are right, the default controller already implements that protocol so there's no need for that.